lib/sync-metrics.js

```diff
@@ -148,28 +148,33 @@ var getStats = function(cb) {
       return input.toFixed(2) + 'ms';
     }
   }];
-  async.map(metricsToFetch, function(metric, callback) {
-    var metricName = metric.metricName;
-    redisClient.lrange([statsNamespace, metricName].join(':'), 0, MAX_NUMBER, function(err, data) {
+  if (redisClient) {
```

Comment:
this change will work, but I think the code will be cleaner if we do it this way:

```
var getStats = function(cb) {
  if(! redisClient) {
    return cb(new Error(....));
  }
  //the rest of code that need no changes
}

```
WDYT?